CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Breaking
 
+- Detect when a user has logged in by means other than JWT, such as ZMI `Basic`
+  authentication or the classic HTML Plone `@login` view @rpatterson
```

Comment:
Is this valid, in the context of Volto?

Reply:
[Ditto](https://github.com/plone/volto/pull/2959#discussion_r814652565)